app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java

```diff
@@ -335,8 +335,10 @@ public void refreshView(State state) {
             ingredientImagePromptLayout.setVisibility(View.VISIBLE);
         }
 
-        if (isBlank(product.getImageNutritionUrl())) {
-            nutritionImagePromptLayout.setVisibility(View.VISIBLE);
+        if(!BuildConfig.FLAVOR.equals( "obf" )) {
```

Comment:
The condition will not work for Open Products Facts (it doesn't have nutrition either)

Reply:
yeah my bad. I'll fix it.